e2e-tests/content-strings/pages/insurance/declarations/modern-slavery/index.js

```diff
@@ -11,4 +11,85 @@ export const MODERN_SLAVERY = {
     },
     IF_YOU_SAY_NO: 'If you say no to any of the questions you must explain why.',
   },
+  EXPANDABLE: {
+    VERSIONS: [
+      {
+        VERSION: '1',
+        INTRO: 'Definition of terms in this declaration',
+        TABLE: {
+          HEADERS: {
+            TERM: 'Term',
+            DEFINITION: 'Definition',
+          },
+          BODY: [
+            {
+              TERM: 'Holding company',
+              DEFINITION: [{ TEXT: 'Means, in relation to a person, any other person in respect of which it is a Subsidiary.' }],
+            },
+            {
+              TERM: 'Modern slavery',
+              DEFINITION: [
+                {
+                  TEXT: 'Means the recruitment, movement, harbouring or receiving of children, women or men through the use of force, coercion, abuse of vulnerability, deception or other means for the purpose of exploitation and includes holding a person in a position of slavery, servitude forced or compulsory labour, or facilitating their travel with the intention of exploiting them soon after.',
+                },
+              ],
+            },
+            {
+              TERM: 'Subsidiary',
+              DEFINITION: [
+                {
+                  CHILDREN: [
+                    {
+                      TEXT: 'Means a subsidiary within the meaning of ',
+                    },
+                    {
+                      HREF: '#',
```

Comment:
Note: the links need to be confirmed by design/BA. We have a subtask for this, EMS-4047.